snuba/datasets/configuration/template.yaml

```diff
@@ -0,0 +1,41 @@
+version: v1
```

Comment:
read through this as if you didn't build it and answer all the questions you may have